src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -337,6 +337,76 @@ void testExecute() throws IOException {
         assertFalse(tdir.exists(), "Temp area should be gone");
     }
 
+    @Test
+    void testExecuteStream() {
+
+        final byte[] data = "bbb".getBytes();
+
+        final String cmd = "/bin/cat";
+
+        int pstat;
+        final StringBuilder sout = new StringBuilder();
+        final StringBuilder serr = new StringBuilder();
+
+        pstat = this.e.execute(cmd, data);
```

Comment:
Overall the unit tests look great! You may want to also add a test case when the data object is empty since there is a check for empty data objects.

Reply:
Added an empty data object to the tests